apps/api-example/src/nestapp/app.module.ts

```diff
@@ -57,7 +57,8 @@ import {PollModule} from '@wepublish/poll/api'
       path: 'v2',
       cache: 'bounded',
       playground: process.env.NODE_ENV === 'development',
-      allowBatchedHttpRequests: true
+      allowBatchedHttpRequests: true,
+      introspection: true
```

Comment:
Use the setting from the config file. Otherwise it looks good

Reply:
Ah this is about allowBatchedHttpRequests not the introspection i think it's fine with batch. 
Plus this is private API, the decision making and config if at all should be applied on gateway.